pkg/runtime/cli.go

```diff
@@ -32,10 +32,12 @@ func FromFlags() (*DaprRuntime, error) {
 	sentryAddress := flag.String("sentry-address", "", "Address for the Sentry CA service")
 	placementServiceAddress := flag.String("placement-address", "", "Address for the Dapr placement service")
 	allowedOrigins := flag.String("allowed-origins", DefaultAllowedOrigins, "Allowed HTTP origins")
-	enableProfiling := flag.String("enable-profiling", "false", fmt.Sprintf("Enable profiling. default port is %v", DefaultComponentsPath))
+	enableProfiling := flag.String("enable-profiling", "false", fmt.Sprintf("Enable profiling. default is false"))
 	runtimeVersion := flag.Bool("version", false, "prints the runtime version")
 	maxConcurrency := flag.Int("max-concurrency", -1, "controls the concurrency level when forwarding requests to user code")
 	mtlsEnabled := flag.Bool("enable-mtls", false, "Enables automatic mTLS for daprd to daprd communication channels")
+	metricsPort := flag.String("metrics-port", fmt.Sprintf("%v", DefaultMetricsPort), "The port for the metrics server")
```

Comment:
What is the default?

Reply:
It needs to be true only if it is k8s mode.

Reply:
So we think this should be handled differently to the gRPC and HTTP ports which both have a default and rely on the user or the cli to set the port to avoid conflict? i.e. https://github.com/dapr/cli/blob/359e2eb8e36f7a5446af1a4569c94ddf2c30764b/pkg/standalone/run.go#L79

Reply:
> So we think this should be handled differently to the gRPC and HTTP ports which both have a default and rely on the user or the cli to set the port to avoid conflict? i.e. https://github.com/dapr/cli/blob/359e2eb8e36f7a5446af1a4569c94ddf2c30764b/pkg/standalone/run.go#L79

Yeah, we use the free port in cli like the others

Reply:
Just to confirm, we're happy to leave this with a default and then make the change in the CLI to get a free port for standalone?

Reply:
@jjcollinge if you want to leave this as is, please file work item on cli repo to assign free port to metric or disable it. Standalone mode users will be happy.

Reply:
Ok done: https://github.com/dapr/cli/issues/275